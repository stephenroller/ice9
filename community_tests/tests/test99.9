# if bool + - bool -> ; fi									succeed

if true + -false -> 
	write "first"; 
fi

write "second";