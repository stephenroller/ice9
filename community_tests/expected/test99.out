Loading file: generated/test99.tm
first
second
Number of instructions executed = 83
